2017/ru/README.md

```diff
@@ -0,0 +1,5 @@
+# __Russian Translation for Top10 - 2017__
```

Comment:
It's a header anyway. Thus, no need to use underscores here.

Reply:
OK. Removed